gc3libs/template.py

```diff
@@ -42,69 +43,7 @@
 except:
     # use our own implementation, in case `itertools` does not (yet)
     # provide the set-product
-    class SetProductIterator(object):
```

Comment:
This can be removed: `itertools` provides `product` since Py2.6, which is the earliest version we support.

Reply:
Nice!

Reply:
Removed